### PR TITLE
[Banner] add snapshot test for middle length text with large layout margin.

### PR DESCRIPTION
**Pull request by fluxminds**
Prepares a snapshot test to solve https://github.com/material-components/material-components-ios/issues/7959.

The snapshot shows that the banner is not generated with a correct height as the button layout is beyond the banner region slightly.:
image

### CODE REVIEW

File: components/Banner/tests/snapshot/MDCBannerSnapshotTests.m [hunk 1]
```diff
@@ -19,9 +19,12 @@
 #import "MaterialTypographyScheme.h"
 
 static NSString *const kBannerShortText = @"tristique senectus et";
+static NSString *const kBannerMiddleLengthText =
```
Comment (romoore) [inline components/Banner/tests/snapshot/MDCBannerSnapshotTests.m:22]: Tip: What makes these "middle" versus "long"? It looks like the difference is one word?

File: components/Banner/tests/snapshot/MDCBannerSnapshotTests.m [hunk 2]
```diff
@@ -19,9 +19,12 @@
...
+    @"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.";
 static NSString *const kBannerLongText =
     @"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.";
 static const CGFloat kBannerContentPadding = 10.0f;
+static const CGFloat kBannerLargeContentPadding = 30.0f;
```
Comment (romoore) [inline components/Banner/tests/snapshot/MDCBannerSnapshotTests.m:29]: Nit: Please add comments to explain what this is used for.

**Review by romoore — APPROVED** [review]
Please add an Issue for this work.

File: components/Banner/tests/snapshot/MDCBannerSnapshotTests.m (hunk 1 above)
Comment (fluxminds) [reply components/Banner/tests/snapshot/MDCBannerSnapshotTests.m:22]: The difference was one word. The bug happens on an edge case, and I didn't want to change these two contants long and extraLong for diff concerns.

File: components/Banner/tests/snapshot/MDCBannerSnapshotTests.m (hunk 2 above)
Comment (fluxminds) [reply components/Banner/tests/snapshot/MDCBannerSnapshotTests.m:29]: Addressed in 09ce09c952e7ca194c3dcee1a4816819aef2cbbe.
